code/admin/CommentAdmin.php

```diff
@@ -48,36 +48,46 @@ public function getEditForm($id = null, $fields = null) {
 
 		$commentsConfig = CommentsGridFieldConfig::create();
 
-		$needs = new GridField(
-			'Comments', 
-			_t('CommentsAdmin.NeedsModeration', 'Needs Moderation'), 
-			Comment::get()->filter('Moderated',0),
+		$new = new GridField(
+			'NewComments',
+			_t('CommentsAdmin.NewComments', 'New Comments'),
+			Comment::get()->filter('IsModerated', 0),
```

Comment:
If we stick to using the `Moderated` column then all old comments will be compatible with the new code. Renaming a column often breaks upgradability.